api/controllers/TelevisionController.js

```diff
@@ -0,0 +1,328 @@
+ * @apiParam {Integer} [room] Room ID 
+ * @apiDescription You can put the devicetype attribute OR the room attribute to determine in which room you want to control the television/which precise deviceType is playing.
+ */
+
+module.exports = {
+   
+
+   /**
+     * @api {post} /television/switchState switch state
+     * @apiName televisionSwitchState
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     */
+    switchState: function(req, res, next){
+        gladys.television.switchState(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/state Get Current State
+     * @apiName televisionGetCurrentState
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {Boolean} state State of the television
+     */
+    getState: function(req, res, next){
+        gladys.television.getState(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/channel Set channel
+     * @apiName televisionSetChannel
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     */
+    setChannel: function(req, res, next){
+        gladys.television.setChannel(req.body)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/getChannel Get channel
+     * @apiName televisionGetChannel
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     *
+     * @apiSuccess {Integer} channel Current channel of the television
+     * 
+     */
+    getChannel: function(req, res, next){
+        gladys.television.getChannel(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/muted Get Muted
+     * @apiName televisionGetMuted
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {boolean} muted Returns true if the television is muted
+     */
+    getMuted: function(req, res, next){
+        gladys.television.getMuted(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {get} /television/volume Get Volume
+     * @apiName televisionGetVolume
+     * @apiGroup television
+     * @apiPermission authenticated
+     *
+     * @apiUse televisionParam
+     * 
+     * @apiSuccess {integer} volume Volume of the television
+     */
+    getVolume: function(req, res, next){
+        gladys.television.getVolume(req.query)
+            .then((result) => res.json(result))
+            .catch(next);
+    },
+
+    /**
+     * @api {post} /television/fastForward Fast Forward 
```

Comment:
Please specify your API in lower case!

Reply:
Fix this on commit c62f1694228a420b53efebcb015d877208a3c3d3